torch/quantization/custom_module_class_mappings.py

```diff
@@ -0,0 +1,55 @@
+OBSERVED_CUSTOM_MODULE_CLASS_MAPPINGS = dict()
+
+def register_observed_custom_module_mapping(custom_module_class, observed_custom_module_class):
+    """ Register a mapping from `custom_module_class` to
+    `observed_custom_module_class`
+    This will be used in prepare step of post training static quantization or
+    quantization aware training
+    """
+    assert hasattr(observed_custom_module_class, 'from_float'), 'from_float must be' + \
+        ' defined in observed custom module class'
```

Comment:
Would be good to provide some details on what is expected in the `from_float` function as well for the Observerd class. Maybe in the docstrings.

Reply:
sure